### DISCUSSION TITLE
Use fallback font without warnings.

### DISCUSSION THREAD

**Discussion by SeanSelene** [wezterm/wezterm#6457] [Q&A]
### What Operating System(s) are you running on?

Windows, macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Here are part of my configuration that related to font:
```lua
return {
  -- --snip--
  font = wezterm.font_with_fallback({
    "JetBrainsMono Nerd Font",
    "FiraCode Nerd Font",
  }),
}
```
If `JetBrainsMono Nerd Font` haven't been installed on my device, wezterm will spawn two windows. One uses `FiraCode Nerd Font` and the the other shows the warnning: 
```shell
"Unable to load a font specified by your font=wezterm.font('JetBrainsMono Nerd
Font1', {weight="Regular", stretch='Normal', style=Normal}) configuration.
Fallback(s) are being used instead, and the terminal may not render as
intended. See https://wezfurlong.org/wezterm/config/fonts.html for more
information"
```
How to let the warnning not shown.

**Comment by accx12** [comment]
How can I suppress the font-related warning in WezTerm when the primary font is not installed, and fallback fonts are being used instead [instagram font generator](https://igfontgen.com)

**Reply by bew** [reply]
You can use [`warn_about_missing_glyphs = false`](https://wezfurlong.org/wezterm/config/lua/config/warn_about_missing_glyphs.html)

**Comment by k1-ship** [comment]
Fallback fonts usually handle missing primary fonts well, but those repeated warnings can get annoying. I’ve seen similar issues while testing an [tamil text generator](http://stylishtamilfonts.com/) with different font setups.
